code/modules/shuttle/shuttle.dm

```diff
@@ -497,14 +497,12 @@
 	// Loop over mobs
 	for(var/t in return_turfs())
 		var/turf/T = t
-		for(var/mob/living/M in T.GetAllContents())
-			// If they have a mind and they're not in the brig, they escaped
-//			if(M.mind && !istype(t, /turf/open/floor/plasteel/shuttle/red) && !istype(t, /turf/open/floor/mineral/plastitanium/red/brig))
-//				M.mind.force_escaped = TRUE
+		for(var/mob/living/L in T.GetAllContents())
 			// Ghostize them and put them in nullspace stasis (for stat & possession checks)
-			M.notransform = TRUE
-			M.ghostize(FALSE)
-			M.moveToNullspace()
+			L.notransform = TRUE
+			var/mob/dead/observer/O = L.ghostize(FALSE)
+			O.timeofdeath = world.time
```

Comment:
Do an `if(O)` because if they are living mobs without a player this will runtime.